packages/fiori-elements-writer/test/__snapshots__/feop.test.ts.snap

```diff
@@ -0,0 +1,3611 @@
+                // call the base component's init function
+                UIComponent.prototype.init.apply(this, arguments);
+
+                // enable routing
+                this.getRouter().initialize();
+            },
+
+            /**
+             * Gets the component startup parameters, setting preferredMode to 'create'.
+             * @public
+             * @returns 
+             */
+            getStartupParameters: function() {
+                return Promise.resolve({
+                    preferredMode: [\\"create\\"]
+                });
+            }
+        });
+    }
+);",
+    "state": "modified",
+  },
+  "webapp/annotations/annotation.xml": Object {
+    "contents": "<edmx:Edmx xmlns:edmx=\\"http://docs.oasis-open.org/odata/ns/edmx\\" Version=\\"4.0\\">
+    <edmx:Reference Uri=\\"https://sap.github.io/odata-vocabularies/vocabularies/Common.xml\\">
+        <edmx:Include Namespace=\\"com.sap.vocabularies.Common.v1\\" Alias=\\"Common\\"/>
+    </edmx:Reference>
+    <edmx:Reference Uri=\\"https://sap.github.io/odata-vocabularies/vocabularies/UI.xml\\">
+        <edmx:Include Namespace=\\"com.sap.vocabularies.UI.v1\\" Alias=\\"UI\\"/>
+    </edmx:Reference>
+    <edmx:Reference Uri=\\"https://sap.github.io/odata-vocabularies/vocabularies/Communication.xml\\">
+        <edmx:Include Namespace=\\"com.sap.vocabularies.Communication.v1\\" Alias=\\"Communication\\"/>
+    </edmx:Reference>
+    <edmx:Reference Uri=\\"sap/opu/odata4/dmo/sb_travel_mduu_o4/srvd/dmo/sd_travel_mduu/0001/$metadata\\">
+        <edmx:Include Namespace=\\"com.sap.gateway.srvd.dmo.sd_travel_mduu.v0001\\" Alias=\\"SAP__self\\"/>
+    </edmx:Reference>
+    <edmx:DataServices>
+        <Schema xmlns=\\"http://docs.oasis-open.org/odata/ns/edm\\" Namespace=\\"local\\">
+        </Schema>
+    </edmx:DataServices>
+</edmx:Edmx>",
+    "state": "modified",
+  },
+  "webapp/i18n/i18n.properties": Object {
+    "contents": "# This is the resource bundle for fefeop1
+
+#Texts for manifest.json
+
+#XTIT: Application name
+appTitle=App \\"Title\\" \\\\\\"
+
+#YDES: Application description
+appDescription=A Fiori application.",
+    "state": "modified",
+  },
+  "webapp/index.html": Object {
+    "contents": "<!DOCTYPE html>
+<html>
+<head>
+    <meta charset=\\"UTF-8\\">
+    <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\">
+    <meta http-equiv=\\"X-UA-Compatible\\" content=\\"IE=edge\\">
+    <title>App &#34;Title&#34; \\\\&#34;</title>
+    <style>
+        html, body, body > div, #container, #container-uiarea {
+            height: 100%;
+        }
+    </style>
+    <script
+        id=\\"sap-ui-bootstrap\\"
+        src=\\"resources/sap-ui-core.js\\"
+        data-sap-ui-theme=\\"sap_belize\\"
+        data-sap-ui-resourceroots='{
+            \\"fefeop1\\": \\"./\\"
+        }'
+        data-sap-ui-compatVersion=\\"edge\\"
+        data-sap-ui-async=\\"true\\"
+        data-sap-ui-frameOptions=\\"trusted\\"
```

Comment:
This would seem the most correct option for the index.html file looking at the documentation: https://sapui5.hana.ondemand.com/#/topic/62d9c4d8f5ad49aa914624af9551beb7